submission_runner.py

```diff
@@ -116,17 +113,16 @@
     enum_values=['jax', 'pytorch'],
     help='Whether to use Jax or Pytorch for the submission. Controls among '
     'other things if the Jax or Numpy RNG library is used for RNG.')
-flags.DEFINE_boolean('profile', False, 'Whether to produce profiling output.')
-flags.DEFINE_boolean('wandb',
-                     False,
-                     'Whether to monitor the results with Wandb.')
-
-flags.DEFINE_string('summary_log_dir',
-                    '',
-                    'Location to dump tensorboard summaries.')
 flags.DEFINE_string('tokenizer_vocab_path',
                     '',
                     'Location to read tokenizer from.')
+
+flags.DEFINE_string('experiment_dir',
+                    None,
+                    'The root directory to store all experiments')
```

Comment:
I would specify here that while not required, the dir should be an absolute dir because otherwise it will be relative to wherever the user is running the script from